server/src/test/java/org/opensearch/action/support/master/TransportMasterNodeActionUtils.java

```diff
@@ -43,12 +43,12 @@ public class TransportMasterNodeActionUtils {
      * a protected method.
      */
     public static <Request extends MasterNodeRequest<Request>, Response extends ActionResponse> void runMasterOperation(
```

Comment:
runClusterManagerOperation

Reply:
Good catch, I will rename the internal method name. 👍 Thanks!

Reply:
Updated in the commit https://github.com/opensearch-project/OpenSearch/pull/2520/commits/a87cede9b88976e0a043e7ea8670d3de2f400c1e